meta-balena-common/recipes-containers/docker-disk/docker-disk.inc

```diff
@@ -34,19 +34,22 @@ python () {
         bb.fatal("docker-disk: Docker needs to run on your host and current user must be able to use it.")
 }
 
+HOSTAPP_PLATFORM_armv5 = "linux/arm/v5"
+HOSTAPP_PLATFORM_armv6 = "linux/arm/v5"
+HOSTAPP_PLATFORM_armv7a = "linux/arm/v7"
+HOSTAPP_PLATFORM_armv7ve = "linux/arm/v7"
+HOSTAPP_PLATFORM_aarch64 = "linux/arm64"
```

Comment:
to be consistent with https://github.com/balena-io-modules/resin-multibuild/blob/master/lib/resolve.ts#L127 I would specify this as `linux/arm64/v8` I don't think it matters though since it's normalized later on by the engine anyway

Reply:
@robertgzr this commit is only renaming the variable. I will do a separate commit to change the value, but I'll do it independently as it has nothing to do with hostapp extensions.

Reply:
https://github.com/balena-os/meta-balena/tree/alexgg/hostapp-platform